third_party/2and3/google/protobuf/symbol_database.pyi

```diff
@@ -1,14 +1,16 @@
 from typing import Dict, Iterable, Type
 
-from .descriptor import EnumDescriptor, FileDescriptor
-from .message import Message
-from .message_factory import MessageFactory
+from google.protobuf.descriptor import Descriptor, EnumDescriptor, FileDescriptor, ServiceDescriptor
+from google.protobuf.message import Message
+from google.protobuf.message_factory import MessageFactory
 
 class SymbolDatabase(MessageFactory):
-    def RegisterMessage(self, message: Type[Message]) -> Type[Message]: ...
-    def RegisterEnumDescriptor(self, enum_descriptor: Type[EnumDescriptor]) -> EnumDescriptor: ...
-    def RegisterFileDescriptor(self, file_descriptor: Type[FileDescriptor]) -> FileDescriptor: ...
-    def GetSymbol(self, symbol: bytes) -> Type[Message]: ...
-    def GetMessages(self, files: Iterable[bytes]) -> Dict[bytes, Type[Message]]: ...
+    def RegisterMessage(self, message: Message) -> Message: ...
```

Comment:
Looking at the docstring, this should indeed be a `Message` subclass, so `Type[Message]` as it was before seems right to me, but I have no experience with protobuf.

Reply:
on closer inspection, it seems the method takes both instances and class objects, I'll fix it